crypto/rand/rand_lcl.h

```diff
@@ -17,21 +17,24 @@
 # include <openssl/ec.h>
 # include "internal/rand.h"
 
-/* we require 256 bits of randomness */
-# define RANDOMNESS_NEEDED (256 / 8)
+/* Amount of randomness (in bytes) we want for initial seeding. */
+# define RANDOMNESS_NEEDED              (128 / 8)
```

Comment:
It's the opinion of Paul (expert) and me (amateur, but who can evaluate other expert sources) that this is all we need.